reversal/client_test.go

```diff
@@ -1,21 +1,24 @@
 package reversal
 
 import (
-	"fmt"
-	"testing"
+	//"fmt"
+	//"testing"
 
 	stripe "github.com/stripe/stripe-go"
-	"github.com/stripe/stripe-go/charge"
-	"github.com/stripe/stripe-go/currency"
-	"github.com/stripe/stripe-go/recipient"
-	"github.com/stripe/stripe-go/transfer"
+	//"github.com/stripe/stripe-go/charge"
+	//"github.com/stripe/stripe-go/currency"
+	//"github.com/stripe/stripe-go/recipient"
+	//"github.com/stripe/stripe-go/transfer"
 	. "github.com/stripe/stripe-go/utils"
 )
 
 func init() {
 	stripe.Key = GetTestKey()
 }
 
+// These tests need to be refactored to no longer used recipients. Create
+// transfers for a target account instead.
+/*
```

Comment:
Yeah it's not great but I think it's okay for now, we will soon delete Recipients from the lib completely I think